docs/sources/api/_index.md

```diff
@@ -151,6 +151,8 @@ And `<stream value>` is:
 }
 ```
 
+The items in the `values` array are sorted by their timestamp in a descending (when using `direction=backward`) or ascending (when using `direction=forward`) order.
```

Comment:
I'm suggesting a rewrite of this to try to be more clear about ordering. It should also make any future localization (translations to other languages) easier.

```suggestion
The items in the `values` array are sorted by timestamp.
The most recent item is first when using `direction=backward`.
The oldest item is first when using `direction=forward`.
```

Reply:
done!